src/base/octets.rs

```diff
@@ -374,17 +392,33 @@ where
     }
 }
 
-#[cfg(features = "smallvec")]
+#[cfg(feature = "smallvec")]
```

Comment:
Interesting. Did the smallvec support work as intended, before? Should I fix it here or revert this change and open a separate PR?